doc/UserGuide.md

```diff
@@ -38,6 +38,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+## Sorting all persons : `sort`
+Sorts a list of all persons in lexicographical order. <br>
```

Comment:
Since the sort command also displays the resulting list of sorted persons, it would be meaningful to document this behaviour too =).